tests/e2e/default/more-options-menu/online-user.wdio-spec.js

```diff
@@ -0,0 +1,119 @@
+
+const sendMessage = async (message = 'Testing', phone = contact.phone) => {
+  await utils.request({
+    method: 'POST',
+    path: '/api/v2/records',
+    headers: {
+      'Content-type': 'application/x-www-form-urlencoded'
+    },
+    body:`message=${message}&from=${phone}`,
+  });  
+};
+
+describe('Online User', async () => {  
+  before(async () => {
+    await loginPage.cookieLogin();
+  });
+  describe('Options disabled when no items: messages, contacts, people', async () => {
+    it('- Message tab', async () => {
+      await commonPage.goToMessages();
+      await commonPage.openMoreOptionsMenu();
+      expect(await commonPage.isOptionEnabled('export', 'messages')).to.be.false;    
+    });
+
+    it(' - Contact tab', async () => {
+      await commonPage.goToPeople();
+      await commonPage.openMoreOptionsMenu();
+      expect(await commonPage.isOptionEnabled('export', 'contacts')).to.be.false;
+      expect(await commonPage.isOptionVisible('edit', 'contacts')).to.be.false;
+      expect(await commonPage.isOptionVisible('delete', 'contacts')).to.be.false;     
+    });
+
+    it('- Report tab', async () => {
+      await commonPage.goToReports();
+      await commonPage.openMoreOptionsMenu();
+      expect(await commonPage.isOptionEnabled('export', 'reports')).to.be.false;
+      expect(await commonPage.isOptionVisible('edit', 'reports')).to.be.false;
+      expect(await commonPage.isOptionVisible('delete', 'reports')).to.be.false;     
+    });
+  });
+
+  describe(' - Options enabled when there are items: messages, contacts, peope', async () => {
+    before(async () => {
+      await utils.saveDocs([ ...places.values(), contact, patient, report ]);
+      await sendMessage();    
+    });
+
+    it('- Contact tab: no contact selected', async () => {
+      await commonPage.goToPeople();
+      await commonPage.openMoreOptionsMenu();
+      expect(await commonPage.isOptionEnabled('export', 'contacts')).to.be.true;
+      expect(await commonPage.isOptionVisible('edit', 'contacts')).to.be.false;
+      expect(await commonPage.isOptionVisible('delete', 'contacts')).to.be.false;
+    });
+
+    it(' - Contact Tab : contact selected', async () => {
+      await commonPage.goToPeople();
+      await contactPage.selectLHSRowByText(contact.name);
+      await (await contactPage.contentRow()).waitForDisplayed();
+      await (await contactPage.contentRow()).click();
+      await contactPage.waitForContactLoaded();
+      await commonPage.openMoreOptionsMenu();
+      expect(await commonPage.isOptionEnabled('export', 'contacts')).to.be.true;
+      expect(await commonPage.isOptionEnabled('edit', 'contacts')).to.be.true;
+      expect(await commonPage.isOptionEnabled('delete', 'contacts')).to.be.true;
+    });
+
+    it('- options enabled when report selected', async () => {
+      await commonPage.goToReports();
+      (await reportPage.firstReport()).click();
```

Comment:
Same please:

Is it selecting a XML or a No XML report?
Because since you added both before, how do we know which one it is testing?

We should test both and check the display of the `Edit`